MapboxVisionAR/Navigation/ARRoute+DirectionsRoute.swift

```diff
@@ -1,3 +1,4 @@
+#if canImport(MapboxDirections)
```

Comment:
`canImport` doesn’t work with CocoaPods: https://github.com/mapbox/mapbox-navigation-ios/issues/1839#issuecomment-438789900.

Reply:
yeah, we decided to remove it because of that